blueocean-dashboard/src/main/js/creation/TempUtils.js

```diff
@@ -0,0 +1,18 @@
+
+function cleanSlashes(url, trailingSlash = true) {
```

Comment:
Yeah, I will split that off in a small PR and get it merged. Sometimes when I'm developing I just copy stuff locally to speed up development time and avoid npm pain. Will get this cleaned up.